From d6d8b5c38cfba4f314e05cb91748750b707be94d Mon Sep 17 00:00:00 2001
From: Jun Li <r65092@freescale.com>
Date: Thu, 15 Jan 2009 15:44:34 +0800
Subject: [PATCH] ENGR00105121 change USB power budget to be 500mA.

Change all USB ports power budget to be 500mA for:
i.MX31 OTG, Host1 and Host2.
i.MX35 OTG and Host2.
i.MX51 OTG and Host1.

Signed-off-by: Li Jun <r65092@freescale.com>
---
 arch/arm/mach-mx3/usb_dr.c  |    4 ++--
 arch/arm/mach-mx3/usb_h1.c  |    4 ++--
 arch/arm/mach-mx3/usb_h2.c  |    4 ++--
 arch/arm/mach-mx35/usb_dr.c |    4 ++--
 arch/arm/mach-mx35/usb_h2.c |    4 ++--
 arch/arm/mach-mx51/usb_dr.c |    4 ++--
 arch/arm/mach-mx51/usb_h1.c |    4 ++--
 7 files changed, 14 insertions(+), 14 deletions(-)

diff --git a/arch/arm/mach-mx3/usb_dr.c b/arch/arm/mach-mx3/usb_dr.c
index 4d8cf17..75d6e20 100644
--- a/arch/arm/mach-mx3/usb_dr.c
+++ b/arch/arm/mach-mx3/usb_dr.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2005-2008 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2005-2009 Freescale Semiconductor, Inc. All Rights Reserved.
  */
 
 /*
@@ -28,7 +28,7 @@ static struct fsl_usb2_platform_data __maybe_unused dr_13783_config = {
 	.platform_init     = usbotg_init,
 	.platform_uninit   = usbotg_uninit,
 	.phy_mode          = FSL_USB2_PHY_SERIAL,
-	.power_budget      = 150,		/* 150 mA max power */
+	.power_budget      = 500,		/* 500 mA max power */
 	.gpio_usb_active   = gpio_usbotg_fs_active,
 	.gpio_usb_inactive = gpio_usbotg_fs_inactive,
 	.transceiver       = "mc13783",
diff --git a/arch/arm/mach-mx3/usb_h1.c b/arch/arm/mach-mx3/usb_h1.c
index 42395e4..5f1d730 100644
--- a/arch/arm/mach-mx3/usb_h1.c
+++ b/arch/arm/mach-mx3/usb_h1.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2005-2008 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2005-2009 Freescale Semiconductor, Inc. All Rights Reserved.
  */
 
 /*
@@ -24,7 +24,7 @@ static struct fsl_usb2_platform_data usbh1_config = {
 	.platform_uninit   = fsl_usb_host_uninit,
 	.operating_mode    = FSL_USB2_MPH_HOST,
 	.phy_mode          = FSL_USB2_PHY_SERIAL,
-	.power_budget      = 150,		/* 150 mA max power */
+	.power_budget      = 500,		/* 500 mA max power */
 	.gpio_usb_active   = gpio_usbh1_active,
 	.gpio_usb_inactive = gpio_usbh1_inactive,
 	.transceiver       = "serial",
diff --git a/arch/arm/mach-mx3/usb_h2.c b/arch/arm/mach-mx3/usb_h2.c
index 6e8d41a..c407e48 100644
--- a/arch/arm/mach-mx3/usb_h2.c
+++ b/arch/arm/mach-mx3/usb_h2.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2005-2008 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2005-2009 Freescale Semiconductor, Inc. All Rights Reserved.
  */
 
 /*
@@ -26,7 +26,7 @@ static struct fsl_usb2_platform_data usbh2_config = {
 	.platform_uninit   = fsl_usb_host_uninit,
 	.operating_mode    = FSL_USB2_MPH_HOST,
 	.phy_mode          = FSL_USB2_PHY_ULPI,
-	.power_budget      = 150,		/* 150 mA max power */
+	.power_budget      = 500,		/* 500 mA max power */
 	.gpio_usb_active   = gpio_usbh2_active,
 	.gpio_usb_inactive = gpio_usbh2_inactive,
 	.transceiver       = "isp1504",
diff --git a/arch/arm/mach-mx35/usb_dr.c b/arch/arm/mach-mx35/usb_dr.c
index 205543f..0b480d3 100644
--- a/arch/arm/mach-mx35/usb_dr.c
+++ b/arch/arm/mach-mx35/usb_dr.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2005-2008 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2005-2009 Freescale Semiconductor, Inc. All Rights Reserved.
  */
 
 /*
@@ -28,7 +28,7 @@ static struct fsl_usb2_platform_data __maybe_unused dr_utmi_config = {
 	.platform_init     = usbotg_init,
 	.platform_uninit   = usbotg_uninit,
 	.phy_mode          = FSL_USB2_PHY_UTMI_WIDE,
-	.power_budget      = 250, /* 250 mA max power */
+	.power_budget      = 500, /* 500 mA max power */
 	.gpio_usb_active   = gpio_usbotg_utmi_active,
 	.gpio_usb_inactive = gpio_usbotg_utmi_inactive,
 	.transceiver       = "utmi",
diff --git a/arch/arm/mach-mx35/usb_h2.c b/arch/arm/mach-mx35/usb_h2.c
index 0487cd1..8e7b705 100644
--- a/arch/arm/mach-mx35/usb_h2.c
+++ b/arch/arm/mach-mx35/usb_h2.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2005-2008 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2005-2009 Freescale Semiconductor, Inc. All Rights Reserved.
  */
 
 /*
@@ -26,7 +26,7 @@ static struct fsl_usb2_platform_data usbh2_config = {
 	.platform_uninit   = fsl_usb_host_uninit,
 	.operating_mode    = FSL_USB2_MPH_HOST,
 	.phy_mode          = FSL_USB2_PHY_SERIAL,
-	.power_budget      = 150,		/* 150 mA max power */
+	.power_budget      = 500,		/* 500 mA max power */
 	.gpio_usb_active   = gpio_usbh2_active,
 	.gpio_usb_inactive = gpio_usbh2_inactive,
 	.transceiver       = "serial",
diff --git a/arch/arm/mach-mx51/usb_dr.c b/arch/arm/mach-mx51/usb_dr.c
index 036bd51..b45ee65 100644
--- a/arch/arm/mach-mx51/usb_dr.c
+++ b/arch/arm/mach-mx51/usb_dr.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2005-2008 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2005-2009 Freescale Semiconductor, Inc. All Rights Reserved.
  */
 
 /*
@@ -29,7 +29,7 @@ static struct fsl_usb2_platform_data __maybe_unused dr_utmi_config = {
 	.platform_init     = usbotg_init_ext,
 	.platform_uninit   = usbotg_uninit_ext,
 	.phy_mode          = FSL_USB2_PHY_UTMI_WIDE,
-	.power_budget      = 150,		/* 150 mA max power */
+	.power_budget      = 500,		/* 500 mA max power */
 	.gpio_usb_active   = gpio_usbotg_hs_active,
 	.gpio_usb_inactive = gpio_usbotg_hs_inactive,
 	.transceiver       = "utmi",
diff --git a/arch/arm/mach-mx51/usb_h1.c b/arch/arm/mach-mx51/usb_h1.c
index 3cf955d..03b3097 100644
--- a/arch/arm/mach-mx51/usb_h1.c
+++ b/arch/arm/mach-mx51/usb_h1.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2005-2008 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2005-2009 Freescale Semiconductor, Inc. All Rights Reserved.
  */
 
 /*
@@ -24,7 +24,7 @@ static struct fsl_usb2_platform_data usbh1_config = {
 	.platform_uninit = fsl_usb_host_uninit,
 	.operating_mode = FSL_USB2_MPH_HOST,
 	.phy_mode = FSL_USB2_PHY_ULPI,
-	.power_budget = 150,	/* 150 mA max power */
+	.power_budget = 500,	/* 500 mA max power */
 	.gpio_usb_active = gpio_usbh1_active,
 	.gpio_usb_inactive = gpio_usbh1_inactive,
 	.transceiver = "isp1504",
-- 
1.5.4.4

